---
title: Home
layout: main
description: Main lab website page
---

<div id="maintext">
<blockquote>
<p><em>
Our lab is interested in characterizing the genetics underlying human disease risk using sporadic cases, extreme phenotypes and familial disease clusters. We assess the effect of such variation using a combination of genetics, molecular biology, statistics and bioinformatics. Our main interests are inflammatory and autoimmune diseases. 
</em></p>
</blockquote>

<h1>What do we research?</h1>

<h2>Hidradenitis suppurativa</h2>
A primary research focis of the lab hidradenitis suppurativa (HS), a chronic skin disease. The first outbreaks are usually during early adulthood, and result in the formation of multiple abscesses in apocrine gland-rich skin folds. This includes under the arms, beneath the breasts, and in the groin. The molecular etiology is not well understood despite a population prevalence of 1%. We collaborate with  Drs. Peter Nagele, Milan Anadkat, and John Kirby for patient recruitment and clinical research.

<h2>Juvenile dermatomyositis</h2>
Juvenile dermatomyositis is a rare disease characterized by a heliotrope facial rash and weakness in both proximal and trunk muscles. A key component of this disease is damage to microvasculature over time. We are interested in the genetic susceptibility underlying disease risk, and in identifying both disease-activity biomarkers and pathways to target for treatment methodologies. We work with the lab of <a href="https://www.luriechildrens.org/en-us/care-services/find-a-doctor/Pages/Pachman_Lauren_1858.aspx">Dr. Lauren Pachman</a> to leverage their clinical resources into molecular research.

<h2>Systemic sclerosis</h2>
Systemic sclerosis, also known as scleroderma, is a progressive disease that manifests with autoimmune, fibrotic, and vascular features. There is a large degree of heterogeneity among patients with regard to how the disease will progress over time. Patient survival has improved over time, but this disease remains amongst the worst diagnoses in rheumatology. We want help improve the outcome of these patients by 1) defining underlying genetic risk factors for the disease, 2) identifying ways to stratify heterogeneous patient groups into more homogeneous molecular pathologies, and 3) create molecular surrogates for skin scores to measure disease activity over time. This work is possible due to a strong collaboration with the <a href="http://scleroderma.northwestern.edu/">Northwestern University Scleroderma program</a>, and in particular the effort of <a href="http://scleroderma.northwestern.edu/members/profile.html?xid=16719">Dr. John Varga</a>, to collect the many clinical samples necessary for this research.

<h1>Jobs</h1>
<strong>**Currently we have no salary funded positions available**</strong>

<h1>Code</h1>
We use statistics and bioinformatics techniques in our research. For statistical analysis we primarily rely on R, and for scripting and programming we tend to use python and C++. You can locate code that we have found useful on the lab <a href="http://github.com/robersonlab">GitHub</a> site.

<h1>Cores</h1>
Part of the lab effort helps to support some core resources available to the general research community. Please contact us if you have any interest in using these core resources.

<h2>RDCC Genetics Core</h2>
Eli runs the Human Genetics and Bioinformatics component of the Washington University Rheumatic Disease Core Center Genetics Core. We focus on providing general advice, IRB consent consultation, experimental design assistance, data analysis, long-term data storage, and contract lab work. Priority will be given to projects related to Rheumatic disease.

<h2>Thrombomicroangiopathy, thrombotic, and thrombolytic disease core</h2>
This is a core collaboration with the Hematology division. We focus on individual patients with putative clotting / complement disorders, or small cohorts of individuals that may have such disorders. Our typical role is to help provide the design of appropriate pathway oriented capture designs, multiplexed sequencing, genotyping from the NGS data, and interpretation of the detected variants.
